nautobot/users/views.py

```diff
@@ -242,17 +243,34 @@ class SavedViewUIViewSet(
     ObjectListViewMixin,
 ):
     queryset = SavedView.objects.all()
+    form_class = SavedViewForm
     filterset_class = SavedViewFilterSet
     serializer_class = SavedViewSerializer
     table_class = SavedViewTable
     action_buttons = ("export",)
 
+    def alter_queryset(self, request):
+        """
+        Two scenarios we need to handle here:
+        1. User can view all saved views with users.view_savedview permission.
```

Comment:
Wonder if we should still enforce object permissions on `view_savedview`, i.e. admin A can view saved views belonging to users in group A, but not saved views for users in group B?

Reply:
Is this what you are thinking?
```
if user.has_perms(["users.view_savedview"]):
            saved_views = queryset.restrict(user, "view")
else:
            ....
```

Reply:
I think so?